README.md

```diff
@@ -76,24 +76,40 @@ instruct Vagrant to load it.
 #### Use Libvirt as provider
 
 In order to use libvirt as provider you need to set
-the value of `vagrant_provider` variable to `libvirt` inside `env.yaml`.
+the value of `conf.vagrant_provider` variable to `libvirt` inside `env.yaml`.
 Vagrant needs to know that you want to use libvirt and not default VirtualBox,
 then you can use the option `--provider=libvirt`.
 
+#### Select a kubernetes networking plugin (CNI)
+
+The networking plugin is configured by setting `ansible.group_vars.all.kubernetes_network_plugin`
+inside `env.yaml`. The currenlty allowed plugins are `weavenet`, `calico` and `flannel`. It is
+also possible to use `no-cni-plugin`. In this case, only the basic configuration of kubernetes will
+be executed on the master node (no plugin will be configured), while in the minion node no command
+to join the kubernetes cluster will be executed.
+
+#### Show verbose output of Ansible operations
+
+Set `conf.additional_ansible_arguments` to `"-vv"` inside `env.yaml` to configure
+verbose output in Ansible. Note that Ansible output is always saved on the
+`ansible_output.txt` file in the `/vagrant` folder.
+
 ### Cleaning up and re-provisioning
 
 If you want to re-test the initializion of the Kubernetes cluster, you can run
 a specific Vagrant provisioner that doesn't run during the normal
 provisioning phase, and then execute the normal provisioning again:
 
 1. `vagrant provision --provision-with cleanup`
+1. `vagrant provision --provision-with mount-shared`
```

Comment:
This looks like an action we might have to do every time we run the cleanup. Consider using [triggers](https://www.vagrantup.com/docs/triggers/configuration.html) instead.

Reply:
Hi @ferrarimarco i'm not sure if triggers will be as efficient as we need, but I'm trying
I've pushed a new Vagrant file, see at line 424 if I use the regular syntax proposed in all examples on vagrant web site:
```
    host.trigger.after :up do |trig|
        trig.info "AFTER PROVISION"
      end
```
and I try vagrant status i receive a weird error
```
C:\GITs\k8s-p9>vagrant status
C:/GITs/k8s-p9/Vagrantfile:425:in `block (4 levels) in <top (required)>': wrong number of arguments (given 1, expected 0) (ArgumentError)
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.7/gems/vagrant-2.2.7/plugins/kernel_v2/config/trigger.rb:113:in `create_trigger'
```
if I use this syntax it works
```
      host.trigger.after :provision,
        info: "AFTER PROVISION"
```

Reply:
going on with the only syntax that works, and pushed the new Vagrant files that executes a run_remote script, it fails as reported hereafter because after the provisioning the VMs is rebooting and so ssh is not working... so making this solution work would take too much time for me... I've already spent a couple of hours on it :-)

```
==> k8s-master-1.k8s-p9.local: Running action triggers after provision ...
==> k8s-master-1.k8s-p9.local: Running trigger...
==> k8s-master-1.k8s-p9.local: AFTER PROVISION
==> k8s-master-1.k8s-p9.local: Trigger run failed
SSH connection was reset! This usually happens when the machine is
taking too long to reboot. First, try reloading your machine with
`vagrant reload`, since a simple restart sometimes fixes things.
If that doesn't work, destroy your machine and recreate it with
a `vagrant destroy` followed by a `vagrant up`. If that doesn't work,
contact support.
```

Reply:
ok don't worry